privacy_evaluator/metrics/accuracy.py

```diff
@@ -0,0 +1,34 @@
+def accuracy_difference(train_accuracy, test_accuracy):
+    """Calculate the difference between the train and test accuracy.
+
+    The difference is calculated by subtracting the test accuracy from thee train accuracy. The result is a positive
+    number if the train accuracy is higher than the test accuracy and a negative number if the train accuracy is smaller
+    than the test accuracy.
+
+    :param train_accuracy: Accuracy of the train phase.
+    :type train_accuracy: int, float
+    :param test_accuracy: Accuracy of the test phase.
+    :type test_accuracy: int, float
+
+    :return: The difference between the train and test accuracy.
+    :rtype: int, float
+    """
+    return train_accuracy - test_accuracy
+
+
+def accuracy_ratio(train_accuracy, test_accuracy):
```

Comment:
also rename to train_to_test_accuracy_ration or something similar, such that it is clear that you divide train by test

Reply:
This is actually code from team 1 and they fixed it as far as we know.